src/argilla/client/apis/datasets.py

```diff
@@ -332,20 +332,24 @@ def _save_settings(self, dataset: _DatasetApiModel, settings: Settings):
                     json=settings_.dict(),
                 )
 
-    def load_settings(self, name: str) -> Optional[Settings]:
+    def load_settings(self, name: str, workspace: Optional[str] = None) -> Optional[Settings]:
```

Comment:
The `load_settings` needs also passing the workspace if different from `set_worksapce`